src/components/create-vm-dialog/createVmDialogUtils.js

```diff
@@ -153,3 +153,15 @@ export function isDownloadableOs(os) {
         // RHSM API supports only rhel versions >= 8: https://access.redhat.com/management/api/rhsm#/images/listImageDownloadsByVersionArch
         (os.version.localeCompare("8.0", undefined, { numeric: true, sensitivity: 'base' }) >= 0));
 }
+
+export function saveOfflineToken(offlineToken) {
+    return localStorage.setItem("offline-token", offlineToken);
```

Comment:
This localStorage key is shared between all cockpit pages, right? I wonder if we should namespace this a bit. I'd call it at least `rhsm-offline-token` (as tokens are being used in a lot of places) to avoid naming collisions. It's also fine to spell this out as `redhat-subscription-offline-token`.

Please move this to a constant instead of repeating it in the three functions here.